content/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Course Home"
+course_title: "Linear Algebra"
+course_image: "https://ocw.mit.edu/courses/mathematics/18-06-linear-algebra-spring-2010/18-06s10.jpg"
```

Comment:
why is this fully qualified?

Reply:
I needed to pull in an image quickly for mocking up the course home page and this was the easiest way to do it.  I did some experimenting with netlify CMS, and their system for handling media uses either "cloudinary" or "uploadcare" as a CDN, allowing you to connect S3 buckets that way.  Once we have something like that set up, this will be just an S3 link connected in that manner.